src/main/ngapp/accio/src/app/editor/MaskToolEnum.ts

```diff
@@ -0,0 +1,5 @@
+export enum MaskTool {
+  magicWand = 'MAGIC-WAND', 
```

Comment:
The enum values (`magicWand`, etc.) should be in upper case with underscores (e.g. `MAGIC_WAND`).

https://google.github.io/styleguide/jsguide.html#naming-enum-names